### PR TITLE
Composer install after pull

### PR DESCRIPTION
**Pull request by maxim-kravets**
I added script that run installation of composer packages after pull or merge. Check, please.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,14 @@ in feature branch. It gets ticket number from branch name. Branch name should ha
 
     > Symlink: ln -s ~/tools/git-hooks/src/pre-commit/check_working_on_master.sh .git/hooks/pre-commit
 
+### post-merge
+
+* [composer_install.sh](src/post-merge/composer_install.sh) - this script installs composer packages specified in 
+composer.json after pull or
```
Comment (greeflas) [inline README.md]: Please add: composer.json or composer.lock if it presents after...

File: src/pre-commit/php_cs_fixer.sh [hunk 2]
```diff
@@ -8,3 +8,5 @@ then
 fi
 
 vendor/bin/php-cs-fixer fix --diff
+
+git add --all
```
Comment (greeflas) [inline src/pre-commit/php_cs_fixer.sh]: It seems this changes comes from previous PR :) Please, rebase with master branch.

File: src/post-merge/composer_install.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+composer -v > /dev/null 2>&1
```
Comment (greeflas) [inline src/post-merge/composer_install.sh]: Is better to check that command exists

```suggestion
command -v composer > /dev/null 2>&1
```

File: src/post-merge/composer_install.sh [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+COMPOSER=$?
+
+if [[ $COMPOSER -ne 0 ]]; then
```
Comment (greeflas) [inline src/post-merge/composer_install.sh]: Please move `then` keyword to the next line regarding project code style

**Review by greeflas — CHANGES_REQUESTED** [review]

**Comment by maxim-kravets** [comment]
@greeflas i did all requested changes.

**Comment by greeflas** [comment]
@maxim-kravets Thanks for contributing!

**Pull request merged**